### DISCUSSION TITLE
To reach claimed training performance the input resolution scaled down 8X?

### DISCUSSION THREAD

**Discussion by alyashev** [NVlabs/instant-ngp#185] [General]
Checking your code I've noticed that to reach the claimed performance the resolution is downscaled 8X.
Am I right? 
When I've set the scaling to 1 the performance drop linearly (which is nice) but it's not the performance you've claimed.
With res 520p (scale = 1/2) with T = 2**19 after 100K iterations and 50min later I've  got PSNR = 35.6 on FOX.
I've increased the T to 2**21 and it takes 13min to run 10K iteration, after 50K iteration PSNR flats out to 36.6.
Is it correct observation?
Thank you.

**Comment by Tom94** [comment]
Hi there, dynamic resolution is independent from training and only affects the rendering preview.

You're right in that the rendering preview can steal compute from training, so for all measured training times from the paper, we disabled rendering until training is done (like prior methods).

I suppose this can indeed be thought of as infinitely downscaling the resolution.
